docs/book/starter-guide/pipelines/parameters-and-caching.md

```diff
@@ -9,11 +9,11 @@ Machine learning pipelines are rerun many times over throughout their developmen
 ## Parameterizing steps
 
 In order to iterate quickly, one must be able to quickly tweak pipeline runs by changing various 
-parameters of steps within a pipeline.
+parameters for the steps that make up your pipeline.
 
 You can configure your pipelines at runtime in the following ways:
 
-- `BaseParameters`: Runtime configuration passed down as a parameter to step functions.
+- `BaseParameters`: Runtime configuration passed down to steps.
```

Comment:
Not sure about this one, the new description sounds like a description for `StepConfiguration`. 

Perhaps "Step parameters passed as an argument to the step"?

Reply:
Maybe @schustmi has a better suggestion? :D

Reply:
Yeah I'm up for any alternatives. I just didn't want us to talk about 'step functions' because those are something different.